src/Raven.Studio/typescript/widgets/virtualGrid/columnPreviewPlugin.ts

```diff
@@ -7,9 +7,16 @@ import moment = require("moment");
 
 
 class copyFeature implements columnPreviewFeature {
+    decodeHtml(html: string): string {
+        const txt = document.createElement("textarea");
+        txt.innerHTML = html;
+        return txt.value;
+    }
+
     install($tooltip: JQuery, valueProvider: () => any, elementProvider: () => any) {
         $tooltip.on("click", ".copy", () => {
-            copyToClipboard.copy(valueProvider(), "Item has been copied to clipboard");
+            const isEncoded = valueProvider() !== this.decodeHtml(valueProvider());
```

Comment:
it is a bit of duck typing. What if we actually pass info if value is already encoded or no?

Reply:
After deeper analysis, i decided that it was sufficient to add valueToCopy arg in the right place in editDocumentConnectedDocuments.ts